batched_atomic.ts

```diff
@@ -142,7 +142,28 @@ export class BatchedAtomicOperation {
    * {@linkcode Deno.KvMutation}.
    */
   mutate(...mutations: Deno.KvMutation[]): this {
-    return this.#enqueue("mutate", mutations);
+    for (const mutation of mutations) {
+      switch (mutation.type) {
+        case "set":
+          this.set(mutation.key, mutation.value, {
+            expireIn: mutation.expireIn,
+          });
+          break;
+        case "delete":
+          this.delete(mutation.key);
+          break;
+        case "sum":
+          this.sum(mutation.key, mutation.value.value);
+          break;
+        case "max":
+          this.max(mutation.key, mutation.value.value);
+          break;
+        case "min":
+          this.min(mutation.key, mutation.value.value);
+          break;
+      }
+    }
```

Comment:
Should there be a default statement throwing invalid mutations to at runtime instead of relying on TypeScript types to this exhaustive switch statement?